graylog2-server/src/main/java/org/graylog2/indexer/Deflector.java

```diff
@@ -194,7 +194,7 @@ public void cycle() {
     }
 
     private void addDeflectorIndexRange(String newTarget) {
-        final IndexRange deflectorRange = indexRangeService.createForDeflector(newTarget);
+        final IndexRange deflectorRange = indexRangeService.createEmptyRange(newTarget);
```

Comment:
As I understand EmptyRange later treated as "Any time".
It looks a bit confusing to me :)